client/src/components/Footer/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FaGithub } from 'react-icons/fa';
+
+export const Footer = () => {
+  return (
+    <div>
+      <h1>
```

Comment:
Let's make this a <p> tag for now. Each page should only have one h1 tag and that's typically the page title or main heading.

Reply:
Good point. Can't remember why they're h1 tags.